besu/src/main/java/org/hyperledger/besu/cli/subcommands/networkcreate/mapping/TOMLMapperAdapter.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.hyperledger.besu.cli.subcommands.networkcreate.mapping;
+
+import java.net.URL;
+import java.nio.file.Path;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.apache.tuweni.toml.Toml;
+import org.apache.tuweni.toml.TomlParseResult;
+
+// TODO Handle errors
+class TOMLMapperAdapter extends MapperAdapter {
```

Comment:
the implementations of all mapper are really close. maybe you could find a way to have a more generic way to deduplicate code